src/pages/ChatPage/components/VoiceMessage.tsx

```
@@ -0,0 +1,58 @@
+
+  const togglePlay = () => {
+    waveSurferRef.current?.playPause();
+    setIsPlaying((prev) => !prev);
+  };
+
+  return (
+    <div className="flex items-center gap-3 rounded-xl bg-gray-100 p-3">
+      <button
+        onClick={togglePlay}
+        className="flex h-8 w-8 items-center justify-center rounded-full bg-blue-600 text-white"
+      >
```

Biome (2.3.14)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)